dashboard/views.py

```diff
@@ -593,11 +587,10 @@ def is_weight_considered(course_id):
     return value
 
 
-def get_term_dates_for_course(course_id):
-    logger.info(get_term_dates_for_course.__name__)
-    sql = "select a.date_start from course c, academic_terms a where c.id = %(course_id)s and c.term_id=a.id"
-    df = pd.read_sql(sql, conn, params={"course_id": course_id}, parse_dates={'date_start': '%Y-%m-%d'})
-    return df['date_start'].iloc[0]
+def get_course_start_date(course_id):
+    logger.info(get_course_start_date.__name__)
+    course_date_start = Course.objects.all().get(id=course_id).get_course_date_range()[0]
```

Comment:
Does "Course.objects.all()" part means to retrieve all course objects from database, instead of just a "get" with course_id?

Reply:
From a performance perspective, I think Django would treat this query as the same as `Course.objects.get(id=course_id)`; see [Querysets are lazy](https://docs.djangoproject.com/en/2.2/topics/db/queries/#querysets-are-lazy). Either way though, I think what you suggested is more concise. I made the change in the latest commit.

Reply:
Good to know!